### DISCUSSION TITLE
Happy to Announce WhisperClip: One-Click Audio Transcription UI!

### DISCUSSION THREAD

**Discussion by gustavostz** [openai/whisper#2047] [Show and tell]
Hello everyone!

I'm happy to share WhisperClip, a simple UI built in Python that makes using Whisper easier. It's designed to make life easier by automatically transcribing audio recordings and saving the text directly to your clipboard. With just a click of a button, it will record your voice, transcribe it, and save it in your clipboard.

Features:

- One-click audio recording and transcription.
- Automatic saving of transcriptions to the clipboard.
- Utilizes the powerful Whisper model for accurate transcriptions.

You can check out the project here: [WhisperClip GitHub Repository](https://github.com/gustavostz/whisper-clip)

If there's interest in a more user-friendly, executable version, I'd be happy to consider creating one. Just let me know!

**Comment by CheekyFloppy** [comment]
Thank you. Maybe add keyboard shortcuts for handling the recording. Much love.

**Reply by gustavostz** [reply]
Done :). Great advice, thanks.

**Reply by gustavostz** [reply]
For now, the default shortcut is alt+shift+r, but you can change it on config.json
